runtime/webhookserver/config.go

```diff
@@ -107,6 +119,15 @@ func NewServer(config interface{}) (Server, error) {
 	if schematypes.MustMap(localtunnelConfigSchema, config, &c) == nil {
 		return NewLocalTunnel(c.BaseURL)
 	}
+	if schematypes.MustMap(webhooktunnelConfigSchema, config, &c) == nil {
+		authCl := auth.New(credentials)
+		whresp, err := authCl.WebhooktunnelToken()
+		if err != nil {
+			return nil, err
+		}
+		s, err := NewWebhookTunnel(whresp)
```

Comment:
Yeah, the authorizer should call auth using tc creds... that way it'll refresh if it gets a 401, right?